app/controllers/work_packages/auto_completes_controller.rb

```diff
@@ -54,15 +54,15 @@ def index
                       .limit(10)
 
     respond_to do |format|
-      format.html do render layout: false end
-      format.any(:xml, :json) { render request.format.to_sym => wp_hashes_with_string }
+      format.html { render layout: false }
```

Comment:
Ah, I see. I actually like that style guide, but no one ever used it. So, :+1:.

> edit: I believe the heavy normalization work, especially regarding format.x do render y end was done in the Rails 4 branch...

Then I better get rid of my commit. No need to risk conflicts for such a minor change.